src/components/views/elements/AccessibleTooltipButton.tsx

```diff
@@ -29,71 +29,56 @@ interface IProps extends React.ComponentProps<typeof AccessibleButton> {
     alignment?: Alignment;
     onHover?: (hovering: boolean) => void;
     onHideTooltip?(ev: SyntheticEvent): void;
-}
+};
 
-interface IState {
-    hover: boolean;
-}
+const AccessibleTooltipButton = forwardRef(function <T extends keyof JSX.IntrinsicElements>(
```

Comment:
conversion to a functional component at the same time as other changes doesn't make this easy to review. If that could happen in a separate PR, or at least a separate commit, in future, that would save a bunch of time.

Reply:
Fair